runtimes/pythonrt/runner/values.py

```diff
@@ -44,6 +44,25 @@ def wrap(v: Any, unhandled: Callable[[Any], pb.Value] = None) -> pb.Value:
     Namedtuples are wrapped as structs.
     """
 
+    history = history or []
```

Comment:
This is a bug. If you pass empty `history` to `wrap` this will create a new one. Use
```python
history = [] if history is None else history
```